common/navbar.js

```diff
@@ -372,6 +438,12 @@
                                 names: menuObject.names
                             });
                         }
+
+                        var openSubmenues = document.querySelectorAll(".dropdown-menu.show");
```

Comment:
What's the point of this piece of code? and why is it not part of the `toggleMenu` function? 

I'm assuming it's making sure the scrollbar is added by setting the height, but it's also forcing the submenu to scroll to top.

Reply:
That's the intention, it's supposed to reset the height values so new ones can be properly set. I didn't realize this was causing the scroll top bug you mentioned with a nested child of submenus.

This block is defined here for the same reason as above, `$window` is not available in the scope of `toggleMenu`.